cdap-ui/app/tracker/controllers/results-ctrl.js

```diff
@@ -140,8 +140,6 @@ class TrackerResultsController {
     if (params.query === '*') {
       params.sort = 'creation-time desc';
       params.numCursors = 10;
-    } else {
-      params.query = params.query + '*';
```

Comment:
I think we should keep the behavior of search exactly the same both on the home page and in the metadata search.